public/scripts/editor/js/bramble-editor.js

```diff
@@ -38,12 +40,19 @@ define(function(require) {
       });
 
       // Start loading the project files
-      Project.load(makeDetails, host, authenticated, function(err, fileToOpen) {
+      Project.load(project, host, function(err, fileToOpen) {
         if(err) {
           console.error("[Thimble Error]", err);
           return;
         }
 
+        // Initialize the project name UI
+        title = project.title || Project.getAnonymousTitle();
```

Comment:
Don't do this, let's force everyone to use `Project.title()` and have it return the right thing no matter what (logged in or anon).